connectors/src/connectors/google_drive/index.ts

```diff
@@ -247,8 +251,9 @@ export class GoogleDriveConnectorManager extends BaseConnectorManager<null> {
         new Error("Tables view is only supported for read permissions")
       );
     }
+    const parentDriveId = parentInternalId && getDriveId(parentInternalId);
```

Comment:
imo after the migration all things called internalIds should be our connector internal connector ids (prefixed)